dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/DateUtils.java

```diff
@@ -506,6 +506,16 @@ public static TimeZone getTimezone(String timezoneId) {
         return TimeZone.getTimeZone(timezoneId);
     }
 
+    /**
+     * get specify date
+     */
+    public static Date getSpecifyDate(int year, int month, int date) {
```

Comment:
Please, this method is only for test, it's not necessary to add this method to increase maintainers' burden, please just move it into the test.